lib/rails_edge_test/dsl/edge.rb

```diff
@@ -74,6 +74,9 @@ def produce_json(ivar: nil)
       ActiveSupport::JSON::Encoding.escape_html_entities_in_json = false
       if ivar
         value = controller.send(:instance_variable_get, ivar)
+        JSON.pretty_unparse(value.as_json)
```

Comment:
Is `as_json` defined on all objects?

Reply:
yes,  `as_json` is what what we being used by itself before `JSON.pretty_unparse` was added in the previous PR

Reply:
oh... oops, I guess it was `to_json` before.  I'll change it to that...

Reply:
Ah, apparently to_json returns  a string, and as_json returns ruby array/hashes.  I think as_json is correct for this purpose, and is the preferred method to override as of rails 3: http://jonathanjulian.com/2010/04/rails-to_json-or-as_json/